domains/doloramoris.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "amoramicae",
+    "email": "amoramicae@gmail.com"
+  },
+  "record": {
+    "A": "45.140.165.216:1148",
```

Comment:
Invalid A record, it should contain only a an array contain strings of just IPs. You have a port specified in your A record.